manager/kubernetes.go

```diff
@@ -53,6 +54,10 @@ func (m *VolumeManager) PVCreate(name, pvName, fsType, secretNamespace, secretNa
 	if fsType == "" {
 		fsType = "ext4"
 	}
+	if fsType == "xfs" && v.Spec.Size < util.MinimalVolumeSizeXFS {
+		return nil, fmt.Errorf("XFS filesystems with size %d, larger than %d, are not supported", v.Spec.Size,
```

Comment:
Shouldn't "larger" be "smaller", just as above?

Reply:
You are right. I'll fix it here and with a quick one on master as well.

Reply:
Actually @james-munson, I prefer to fix it with https://github.com/longhorn/longhorn-manager/pull/2818, which I will backport. It is cleaner that way since all stable branches will get similar commits. WDYT?